### ISSUE TITLE
Instrumentation not shown in Database transaction tab

### ISSUE THREAD

**Issue by AvnerCohen** [evanphx/newrelic-redis#15]
Looking at the core rpm instrumentation for mongo, I see it is added under the "Datastore" namespace:
https://github.com/newrelic/rpm/blob/master/lib/new_relic/agent/datastores/mongo/metric_translator.rb#L54

While redis is added under "database":

https://github.com/evanphx/newrelic-redis/blob/master/lib/newrelic_redis/instrumentation.rb#L67

I wonder if simply changing the namespace is all that is needed, or there is more to it?

Basically what I want is to have the redis issues pile up in the "Slow SQL Traces" block of the UI.
